godot-macros/src/util.rs

```diff
@@ -431,22 +427,40 @@ mod tests {
     use proc_macro2::TokenStream;
     use quote::quote;
 
-    macro_rules! hash_map {
+    macro_rules! kv_map {
         (
             $($key:expr => $value:expr),*
             $(,)?
         ) => {
             {
                 let mut map = std::collections::HashMap::new();
                 $(
-                    map.insert($key, $value);
+                    map.insert(ident($key), $value);
                 )*
                 map
             }
         };
     }
 
-    fn expect_parsed(input_tokens: TokenStream, output_map: KvMap) {
+    macro_rules! kv_value {
+        ($($args:tt)*) => {
+            KvValue::new(quote!($($args)*).into_iter().collect())
+        }
+    }
+
+    impl PartialEq for KvValue {
+        fn eq(&self, other: &Self) -> bool {
+            let to_strings = |kv: &Self| {
+                kv.tokens
+                    .iter()
+                    .map(ToString::to_string)
+                    .collect::<Vec<String>>()
+            };
+            to_strings(self) == to_strings(other)
+        }
+    }
```

Comment:
I see. But why move it outside if it's only used inside `#[cfg(test)]`?

Reply:
Agree, but in this case it probably makes sense to have "test-only code" together -- since it's not really a trait that can be used elsewhere with the type.